data/categories.txt

```diff
@@ -3005,7 +3005,7 @@ da:Tempel|kultsted
 nl:Tempel|gebedsplaats|gebedshuis
 fi:Temppeli|uskonnollinen kohde
 fr:Temple|lieu de culte
-de:Tempel|Anbetungsstätte
+de:Hinduistischer Tempel|Tempel|Anbetungsstätte
```

Comment:
Can we create a @temple (Tempel) alias and use it in concrete subtypes of temples, to avoid copy-pasting the same "Tempel" everywhere (in a separate PR, of course)?

Reply:
Counted 8 different places of worship, so that should be worth it. I'd rather name it @**place_of_worship** however.